From: Frederic Crozat <fcrozat@suse.com>
Date: Mon, 16 Jun 2014 18:49:12 +0200
Subject: bootchart: set white background

 In programs like eog and gimp the transparant background did not
look very good.

Similar fix from the one done in systemd-analyze (418e3750)

(cherry picked from commit a7997073ac07add6cb30dcd9491c0179b2d6ad52)
(cherry picked from commit 0eca797c3c6b82527d1e38a9f194f9daca1abb47)
---
 src/bootchart/svg.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/bootchart/svg.c b/src/bootchart/svg.c
index 5eee2d19..fe8b1b8a 100644
--- a/src/bootchart/svg.c
+++ b/src/bootchart/svg.c
@@ -123,6 +123,7 @@ static void svg_header(void) {
         svg("<defs>\n  <style type=\"text/css\">\n    <![CDATA[\n");
 
         svg("      rect       { stroke-width: 1; }\n");
+        svg("      rect.bg    { fill: rgb(255,255,255); }\n");
         svg("      rect.cpu   { fill: rgb(64,64,240); stroke-width: 0; fill-opacity: 0.7; }\n");
         svg("      rect.wait  { fill: rgb(240,240,0); stroke-width: 0; fill-opacity: 0.7; }\n");
         svg("      rect.bi    { fill: rgb(240,128,128); stroke-width: 0; fill-opacity: 0.7; }\n");
@@ -1276,6 +1277,7 @@ void svg_do(const char *build) {
 
         /* after this, we can draw the header with proper sizing */
         svg_header();
+        svg("<rect class=\"bg\" width=\"100%%\" height=\"100%%\" />\n\n");
 
         svg("<g transform=\"translate(10,400)\">\n");
         svg_io_bi_bar();
